new-test-case.mdx

```diff
@@ -4,9 +4,41 @@ description: "Add more test cases to your test suite. Use AI generation or recor
 icon: "arrow-progress"
 ---
 
+## Have AI agent suggest and auto-generate more tests
+
+Similar to the inital **AI discovery** we can run the discovery feature to "spawn" individual test cases.
+
+Go into the test case and hover over the `suggest more` icon - a button will appear. Click on it.
+
+<Frame caption="Have the AI agent suggest more test cases, 08/2024">
+  <img
+    src="images/setup/setup-11-suggest-more.png"
+    alt="have AI suggest more test cases"
+  />
+</Frame>
+
+AI agent will discover up to 3 new tests following up on the test case you launched it from. The original test will be added as a **dependency** automatically. It will auto-generate steps for every discovered test case and validate them.
```

Comment:
```suggestion
Our AI agent will discover up to 3 new tests following up on the test case you launched it from. The original test will be added as a **dependency** automatically. It will auto-generate steps for every discovered test case and validate them.
```